test/jdk/java/util/Currency/PropertiesTest.sh

```diff
@@ -27,6 +27,7 @@
 #      8157138 8190904
 # @summary tests the capability of replacing the currency data with user
 #     specified currency properties file
+# @requires vm.flagless
```

Comment:
Does this actually do anything? Since it is a shell script, it does not call any `ProcessBuilder` methods. In fact, the script includes `${TESTVMOPTS}` on launching the test java process correctly.

Reply:
IIUC, using `@requires vm.flagless` simply signifies that any JVMs launched within the test do not support VM flags, (regardless if it was launched from ProcessBuilder).

However, it is odd that this test was flagged since it does pass `${TESTVMOPTS}` to `run()` like you stated. 

Perhaps @lmesnik has further understanding of what to do here?

Reply:
_PropertiesTest.sh_ needs to pass both test.vm.opts **and** test.java.opts. Added `${TESTJAVAOPTS}` and unmarked as `@requires vm.flagless`